src/components/Admin/TitleLogoUpload.jsx

```diff
@@ -0,0 +1,133 @@
+import { useTranslation } from "react-i18next";
+import { Buffer } from "buffer";
+import { FileUp } from "lucide-react";
+import { Image } from "next/image";
```

Comment:
@theredwillow the fix to the  react render error is

```suggestion
import Image from "next/image";
```
There is a some detail in the `Image` component that requires it to be an import of "default export"
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

Don't understand all of it, but this fixes the LogoUpload.

If you make this fix I can test everything out and we can move on 👍

Reply:
1556ef83ca50d83ec02a7b4780f9abfbd9b2333e